upgrade_tests/test_existance_relations/cli/test_contenthosts.py

```diff
@@ -29,10 +29,12 @@
 component = 'content-host'
 ch_name = compare_postupgrade(component, 'name')
 ch_errata = compare_postupgrade(component, 'installable errata')
-
+unsupported_versions = ['6.3', '6.4', '6.5', '6.6', '6.7', '6.8']
```

Comment:
This shouldn't be global variable as this is not applicable for all the tests in this module.

New contributor might unintentionally copy the `dont_run` decorator for supported versions test and end up not running the tests.

Reply:
I would recommend to revert to the previous implementation.

Reply:
Thanks, Changes rolled back.